e2e/_suites/ingest-manager/stand-alone.go

```diff
@@ -230,7 +237,7 @@ func searchAgentData(hostname string, startDate time.Time, minimumHitsCount int,
 		},
 	}
 
-	indexName := "logs-agent-default"
+	indexName := ".ds-logs-elastic.agent-default-000001"
```

Comment:
we're not supposed to have to use the .ds in order to find things in a search.  is this used only to assert that a new index is being accessed?   I don't know the Data Streams feature well to be 100% sure of usage just confirming.  How is it used?  We can ask @michalpristas to look at the usage and help confirm...

Reply:
I'd like to know if there is a way to define an alias. For metricbeat we do this: https://github.com/elastic/e2e-testing/blob/a9fe641faf722ac487e9a6423b5bb321ca29d2d2/cli/config/compose/services/metricbeat/docker-compose.yml#L7 We define an index for each metricbeat that is run. Maybe it's not doable in the same way, but I'd love to define an index to be used during the entire test suite.